test/e2e/network/util_iperf.go

```diff
@@ -101,7 +103,7 @@ func (s StrSlice) get(i int) string {
 func intOrFail(debugName string, rawValue string) int64 {
 	value, err := strconv.ParseInt(rawValue, 10, 64)
 	if err != nil {
-		framework.Failf("Failed parsing value %v from the string '%v' as an integer", debugName, rawValue)
+		framework.Failf("ginkgo.Failed parsing value %v from the string '%v' as an integer", debugName, rawValue)
```

Comment:
```suggestion
		framework.Failf("Failed parsing value %v from the string '%v' as an integer", debugName, rawValue)
```

Reply:
Thanks a lot :+1: 
I've updated the code.